lib/controller.js

```diff
@@ -45,152 +33,134 @@ const getEmoji = (repoLength) => {
-exports.index = (req, res) => {
-    const username = req.query.username;
-    const headers = {
-        'User-Agent': `${username}`
-    };
-
-    const userOptions = {
-        url: `https://api.github.com/users/${username}?client_id=${client_id}&client_secret=${client_secret}`,
-        headers
-    };
-
-    const repoOptions = {
-        url: `https://api.github.com/users/${username}/repos?client_id=${client_id}&client_secret=${client_secret}&per_page=100`,
-        headers
-    };
+const reqUser = (username) => {
+    const url = `https://api.github.com/users/${username}?client_id=${client_id}&client_secret=${client_secret}`;
+    const headers = { 'User-Agent': `${username}` };
+    return axios.get(url, { headers });
+};
 
-    let languages = {},
-        repos = '',
-        msg = '',
-        avatar,
-        statement;
+const reqRepos = (username, numberOfPages) => {
+    const headers = { 'User-Agent': `${username}` };
+    const url = (page) =>
+        `https://api.github.com/users/${username}/repos?client_id=${client_id}&client_secret=${client_secret}&per_page=100&page=${page}`;
+    const requests = [];
+    for (let i = 1; i <= numberOfPages; i++) {
+        requests.push(axios.get(url(i), { headers }));
+    }
+    return requests;
+};
 
+exports.index = (req, res) => {
+    const { username } = req.query;
+    let avatar, msg, repos, statement;
+    let languages = {};
+    const statements = [
+        'You\'ve got skills on',
+        'Wow! You really like',
+        'Good job on',
+        'Keep going on'
+    ];
+    const mysteryStatements = [
+        'Wow, you like the mystery language',
+        'Wow, you are a mystery man!'
+    ];
     if (username === undefined) {
         res.render('layouts/main', {
             show: {
                 result: false,
                 chart: false
             },
-            username,
+            username: '',
             avatar,
             languages,
             msg,
             repos,
             statement
         });
-
         return;
     }
-
-    request.get(userOptions, (err, response, body) => {
-        const data = JSON.parse(body);
-
-        if (data.message === 'Not Found') {
-            // user was not found
-
-            msg = 'User was not found';
-
-            res.render('layouts/main', {
-                show: {
-                    language: false,
-                    result: false,
-                    chart: false
-                },
-                username,
-                avatar,
-                languages,
-                msg,
-                repos,
-                statement
-            });
-
-            return;
-        }
-
-        avatar = data.avatar_url;
-
-        // get repo
-        request.get(repoOptions, (err, response, body) => {
-            const dataRepo = JSON.parse(body);
-            let repoLength = dataRepo.length;
-
-            if (repoLength === 0) {
+    reqUser(username)
+        .then((usr) => {
```

Comment:
Just use `user` and `userData` would be good for naming conventions

Reply:
:+1: